common/config/.golangci.yml

```diff
@@ -210,6 +210,8 @@ linters-settings:
             desc: "don't use v3; v2 is orders of magnitude higher performance"
           - pkg: k8s.io/apimachinery/pkg/util/sets
             desc: "use istio.io/istio/pkg/util/sets"
+          - pkg: k8s.io/utils/set
+            desc: "use istio.io/istio/pkg/util/sets"
```

Comment:
nit: why do we have a set fork?

Reply:
Our opinions vs theirs largely. We had better suited for Istio opinions at the time, they didn't adopt things we depended on at the time. Likely stdlib will get set eventually, though we will probably keep our own wrapper like we do for `slices` and `maps` anyways